test/test.dart

```diff
@@ -415,3 +418,20 @@ extension RealmObjectTest on RealmObject {
 }
 
 void clearCachedApps() => realmCore.clearCachedApps();
+
+Future<void> _printPlatformInfo() async {
```

Comment:
Is this info useful when running locally. We print the PID cause its is useful when debugging locally. The other stuff might be only useful on CI but not sure about that. Not sure running uname each time is beneficial especially locally.  So should we be doing this only when REALM_CI is set?

Reply:
The main reason why I wanted to add `uname` is because on macOS it's super easy to run under rosetta accidentally (there's no prompt or anything). So it's kind of hard to determine whether you're testing on arm or x64. Locally, I don't see a meaningful impact of running `uname` but can disable it if it impacts people.